Source/Loader/Utils/SteamUtils.cs

```diff
@@ -91,6 +91,10 @@ public static string GetGameInstallPath(string FolderName)
 
         public static bool IsSteamRunningAndLoggedIn()
         {
+            if (Environment.GetEnvironmentVariable("YES_STEAM_IS_RUNNING") == "1")
```

Comment:
Can you remove this please, as its really just a hack, the actual code here needs to be fixed really.

I think the better solution here is probably just going to be iterating the active processes and see if steam is running.

Reply:
That wont work either as in wine / proton you won't see steam, it runs natively on linux side. What you could do however is load steamapi and check if steam is running through that as valve proxies libsteamapi between wine <-> native

Reply:
Simpler solution might be to check this env variable :thinking:
https://github.com/ValveSoftware/Proton/blob/proton_9.0/steam_helper/steam.cpp#L1624

Reply:
Ok. Well I'll merge the pull request, can follow up on this later.